lib/Cake/Utility/String.php

```diff
@@ -355,6 +355,23 @@ public static function wrap($text, $options = array()) {
  * @return string Formatted text.
  */
 	public static function wordWrap($text, $width = 72, $break = "\n", $cut = false) {
+		$paragraphs = explode($break, $text);
+		foreach ($paragraphs as &$paragraph) {
+			$paragraph = String::_wordWrap($paragraph, $width, $break, $cut);
```

Comment:
Yeah, but it can easily be user-adjusted to be fully justified by removing the newline characters before and STILL allow the custom newlines for other use cases. But as of now it can't do the latter at all. And for the first you would need the manual removal anyway either way.